tdrs-backend/tdpservice/security/clients.py

```diff
@@ -62,21 +69,31 @@ def scan_file(self, file, file_name) -> bool:
                 files={'file': file},
                 timeout=settings.AV_SCAN_TIMEOUT
```

Comment:
It might be my environment, but I was getting an error until I converted this `                timeout=int(settings.AV_SCAN_TIMEOUT)`, thoughts @jtwillis92?

Reply:
Interestingly, I haven't run into that personally. However, those values should definitely be integers so I updated `settings/common.py` to cast them to int in the settings declaration.

Reply:
Hmm and I just had the defaults in my .env 🤷  I think this works.